src/components/copyText/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { Button, Tooltip } from 'antd';
+import styled from 'styled-components';
+
+interface CopyTextProps {
+  readonly text: string;
+  readonly onCopy?: (text: string) => Promise<void>;
+}
+
+export const copyToClipboard = (text: string): Promise<void> => {
+  return navigator.clipboard.writeText(text);
```

Comment:
‼️ Where does navigator come from? It looks like its a window scoped global variable... you know how I feel about those 🤢

Reply:
⛏Would also prefer if you left a comment in the code explaining this a bit more

Reply:
It might be... there's really no way around it though. We could install [react-copy-to-clipboard](https://www.npmjs.com/package/react-copy-to-clipboard), but that uses `window.clipboardData.setData(format, text)` under the hood anyway.